lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -6,10 +6,9 @@ class JsonApi < Base
         autoload :PaginationLinks
         autoload :FragmentCache
         autoload :Link
-        autoload :Association
-        autoload :ResourceIdentifier
         autoload :Meta
         autoload :Deserialization
+        require 'active_model/serializer/adapter/json_api/api_objects'
```

Comment:
So, in https://github.com/rails-api/active_model_serializers/pull/1004/files#r52563837 I noticed that we've already started a pattern of sticking the json_api objects in the json_api folder. e.g. above we have Meta and Link which are also json_api object.

In code I removed I move that object into `json_api/jsonapi.rb`.  What do you think about just sticking also the api objects in the `json_api` folder as we've essentially already been doing?